lib/auth/webauthncli/api.go

```diff
@@ -116,10 +129,10 @@ func Login(
 		return crossPlatformLogin(ctx, origin, assertion, prompt, opts)
```

Comment:
Since the Windows code handles both platform and cross-platform, could we pull the "if winwebauthn.IsAvailable()" before this block and handle all Windows code there?

Reply:
done